modules/vaos/app/services/eps/appointment_service.rb

```diff
@@ -174,4 +185,8 @@ def redis_client
       @redis_client ||= Eps::RedisClient.new
     end
   end
+
+  # Mirrors the middleware-defined EPS exception so callers can rely on
+  # BackendServiceException fields (e.g., original_status, original_body).
+  class ServiceException < Common::Exceptions::BackendServiceException; end unless defined?(Eps::ServiceException)
```

Comment:
added exception definition here to allow for unit testing, exception class is dynamically generated by the middleware leveraged from `Common::Client::Base:perform` upon error response.